gloo/docs/gloo_routing/virtual_services/security/custom_auth/_index.md

```diff
@@ -4,40 +4,77 @@ weight: 80
+the Envoy spec for an [external authorization server](https://github.com/envoyproxy/envoy/blob/master/api/envoy/service/auth/v2/external_auth.proto).
 
-## Deploy Gloo and the petstore demo app
+## Setup
+{{< readfile file="/static/content/setup_notes" markdown="true">}}
 
-Install Gloo-enterprise (version v0.13.5 or above) and the petstore demo:
+Let's start by creating the sample `petstore` application:
 
 ```shell
-glooctl install gateway enterprise --license-key <YOUR KEY>
 kubectl apply --filename https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
 ```
 
-Add a route and test that everything so far works:
+We can now add a route to the sample application by running the following command:
 
-```shell
+{{< tabs >}}
+{{< tab name="kubectl" codelang="shell" >}}
+kubectl apply -f - <<EOF
+apiVersion: gateway.solo.io/v1
+kind: VirtualService
+metadata:
+  name: default
+  namespace: gloo-system
+spec:
+  virtualHost:
+    domains:
+    - '*'
+    routes:
+    - matcher:
+        prefix: /
+      routeAction:
+        single:
+          upstream:
+            name: default-petstore-8080
+            namespace: gloo-system
+EOF
+{{< /tab >}}
+{{< tab name="glooctl" codelang="shell" >}}
 glooctl add route --name default --namespace gloo-system --path-prefix / --dest-name default-petstore-8080 --dest-namespace gloo-system
-curl "$(glooctl proxy url)/api/pets/"
+{{< /tab >}}
+{{< /tabs >}}
+
+Let's verify that everything so far works by querying the virtual service.
+
+```shell script
+curl $GATEWAY_URL/api/pets/
 ```
 
+You should see the following output:
+
 ```json
 [{"id":1,"name":"Dog","status":"available"},{"id":2,"name":"Cat","status":"pending"}]
 ```
 
-## HTTP Authentication service intro
+## Creating a simple HTTP Authentication service
 
-When using an HTTP auth service, the request will be forwarded to the authentication service. If the
-auth service returns `200 OK` it is considered authorized. Otherwise the request is denied.
-You can fine tune which headers are sent to the the auth service, and wether or not the body is forwarded as well, by editing the [extauth extension](/api/github.com/solo-io/solo-projects/projects/gloo/api/v1/plugins/extauth/extauth.proto.sk/#settings) settings in the Gloo settings (see [below](#configure-gloo-settings) for an example of the Gloo settings with the extension settings).
+When a request matches a route that defines an `extauth` configuration, Gloo will forward the request to the external 
+auth service. If the HTTP service returns a `200 OK` response, the request will be considered authorized and sent to 
+its original destination. Otherwise the request will be denied.
+You can fine tune which headers are sent to the the auth service, and whether or not the body is forwarded as well, 
+by editing the [extauth settings]({{< ref "/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/plugins/extauth/v1/extauth.proto.sk#settings" >}}) 
```

Comment:
based on some stuff @mitchdraft is working on, i think the link should be:
```suggestion
by editing the [extauth settings]({{< ref "/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/plugins/extauth/v1/extauth.proto.sk/#settings" >}}) 
```

e.g. https://github.com/solo-io/solo-kit/pull/298/files

Reply:
The additional slash breaks the site for me

Reply:
the best way to do this is `{{< protobuf name="proto.import.name" >}}` I'll add an option to provide a display name as well